engine_manifest.json

```
@@ -54,6 +52,11 @@
             "type": "bool",
             "value": true,
             "name": "2人の話者でモーフィングした音声を合成"
+        },
+        "manage_library": {
+            "type": "bool",
+            "value": false,
+            "name": "ライブラリのインストール・アンインストール"
```

あ、ライブラリだとpipでインストールされるのと混合しちゃいそうなので音声ライブラリにしています！
忘れないうちに。。
```suggestion
            "name": "音声ライブラリのインストール・アンインストール"
```